src/jdk.jdwp.agent/share/native/libjdwp/eventHelper.c

```diff
@@ -594,15 +604,20 @@ handleReportVMInitCommand(JNIEnv* env, ReportVMInitCommand *command)
                           JDWP_COMMAND_SET(Event),
                           JDWP_COMMAND(Event, Composite));
     (void)outStream_writeByte(&out, command->suspendPolicy);
-    (void)outStream_writeInt(&out, 1);   /* Always one component */
-    (void)outStream_writeByte(&out, JDWP_EVENT(VM_INIT));
-    (void)outStream_writeInt(&out, 0);    /* Not in response to an event req. */
+    (void)outStream_writeInt(&out, 1); /* Always one component. */
+    (void)outStream_writeByte(&out, jdwpEvent);
+    (void)outStream_writeInt(&out, 0); /* Not in response to an event request. */
 
     (void)outStream_writeObjectRef(env, &out, command->thread);
 
     outStream_sendCommand(&out);
     outStream_destroy(&out);
     /* Why aren't we tossing this: tossGlobalRef(env, &(command->thread)); */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+    if (JDWP_EVENT(VM_RESTORE) == jdwpEvent) {
+        tossGlobalRef(env, &command->thread);
```

Comment:
Please indent only with spaces.

Reply:
Yeah, it failed the Line Endings Check initially and was fixed afterwards.